### ISSUE TITLE
Toggling dense paragraph OFF

### ISSUE THREAD

**Issue by japanoise** [davidgiven/wordgrinder#62]
Hi

I much prefer the look when dense paragraphs are off - it makes it much easier to read. However, it doesn't seem to be working. I've toggled it off in the menu; which works until I close wordgrinder. I've also tried putting

```lua
GlobalSettings.lookandfeel.denseparagraphs = false
```

into my .wordgrinder.lua, but that seems to do nothing.

How can I turn dense paragraphs off by default?

**Comment by japanoise** [comment]
No, nevermind, it turns out I can call:

```lua
SaveGlobalSettings()                                                  
UpdateDocumentStyles()
```

which seems to propogate the change.

Still curious as to why I can't change it in the menus, though.

**Comment by davidgiven** [comment]
Having just checked, that's a bug at my end --- a recent fix broke saving of the .wordgrinder.settings file. I'll fix. Thanks for reporting this!

**Comment by japanoise** [comment]
Well that seemed to have fixed the issue of the .wordgrinder.settings file not being loaded, which is great! I see the settings in there:

```
.globalSettings.debug.currentword: false
.globalSettings.debug.location: false
.globalSettings.debug.memory: false
.globalSettings.lookandfeel.denseparagraphs: false
.globalSettings.lookandfeel.enabled: true
.globalSettings.lookandfeel.maxwidth: 100
.globalSettings.lookandfeel.terminators: true
.globalSettings.systemdictionary.filename: "/etc/dictionaries-common/words"
```

However, when I start up and load a document, the widescreen mode works but the dense paragraph setting doesn't.

**Comment by japanoise** [comment]
My completely unprofessional opinion is that for whatever reason, `UpdateDocumentStyles` isn't being called, and thus the non-dense paragraphs aren't showing up.

**Comment by davidgiven** [comment]
Hmm. It works fine for me.

- do 'wordgrinder README.wg'
- observe non-dense paragraphs
- change to dense paragraphs
- observe dense paragraphs
- quit but don't save

- do 'wordgrinder README.wg'
- observe dense paragraphs
- check to non-dense paragraphs
- observe non-dense paragraphs
- quit but don't save

Does this match what you're doing?

**Comment by davidgiven** [comment]
Yep, that's completely reproducable and embarrassing. Your completely unprofessional opinion was correct, by the way.

Fixed in 48f8cbfb0fdfc4c5c3bff691114174e8d5705ddc --- thanks very much!

**Comment by japanoise** [comment]
Thank you! It all seems to work perfectly now :) keep it up!
